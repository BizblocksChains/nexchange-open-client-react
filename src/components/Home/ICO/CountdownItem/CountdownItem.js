import React from 'react';
import styles from './CountdownItem.scss';

const CountdownItem = props => (
  <div className={styles.item}>
    <div className={styles.count}>{props.count}</div>
    <div className={styles.period}>{props.period}</div>
  </div>
);

export default CountdownItem;
