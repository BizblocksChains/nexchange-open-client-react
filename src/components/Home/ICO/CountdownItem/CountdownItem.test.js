import React from 'react';
import { shallow } from 'enzyme';
import CountdownItem from './CountdownItem';

describe('CountdownItem', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<CountdownItem period="days" count="1" />);
    expect(wrapShallow).toMatchSnapshot();
  });
});
