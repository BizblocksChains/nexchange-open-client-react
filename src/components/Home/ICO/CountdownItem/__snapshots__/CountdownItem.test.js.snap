// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountdownItem renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CountdownItem
    count="1"
    period="days"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="count"
>
          1
</div>,
        <div
          className="period"
>
          days
</div>,
      ],
      "className": "item",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1",
          "className": "count",
        },
        "ref": null,
        "rendered": "1",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "days",
          "className": "period",
        },
        "ref": null,
        "rendered": "days",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="count"
>
            1
</div>,
          <div
            className="period"
>
            days
</div>,
        ],
        "className": "item",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1",
            "className": "count",
          },
          "ref": null,
          "rendered": "1",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "days",
            "className": "period",
          },
          "ref": null,
          "rendered": "days",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
